torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -622,6 +623,34 @@ Tensor _sparse_addmm_sparse_backward(const Tensor& grad, const Tensor& sparse_,
   return grad_sparse.sparse_mask(sparse);
 }
 
+Tensor _sparse_matrix_mask(const Tensor& grad, const Tensor& mask){
```

Comment:
This seems to do the same as `sparse_mask` function, which is used in the grad tests, no?
https://github.com/pytorch/pytorch/blob/9dc3d8738cafe84f89e36c5358f13ae3355825cc/aten/src/ATen/native/sparse/cuda/SparseCUDATensor.cpp#L60
https://github.com/pytorch/pytorch/blob/edf751ca2fededecdd9366874c761431c0f61f01/aten/src/ATen/native/sparse/SparseTensor.cpp#L522

Reply:
No, it's not exactly the same. This is a mask operation over a dense tensor. What I implemented is a mask operation over a sparse tensor.